NodeJS/Project/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "clientInteractive.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
Just one thing. in node, there is a concept of devDependencies... some libraries that are required only on development time should be added under this propery.

For example, instead of running `npm install mocha --save` you can run `npm install mocha --save-dev`
This way, in the final version of your code, it will not be included in the package, which makes it smaller.

Reply:
Thank you. Going to keep that in mind for next project.